ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -875,6 +875,23 @@ class JITModBuilder {
     if (OldGV->isConstant()) {
       GV->copyAttributesFrom(&*OldGV);
       cloned_globals.push_back(OldGV);
+      // If this is a vector containing other constants, we need to clone
+      // those as well.
+      Type *Ty = OldGV->getValueType();
+      Constant *C = OldGV->getInitializer();
+      if (Ty->isArrayTy()) {
+        if ((cast<ArrayType>(Ty))->getElementType()->isPointerTy()) {
+          auto GVA = dyn_cast<ConstantArray>(C);
```

Comment:
How do we feel about `auto`?

Reply:
[429140f](https://github.com/ykjit/yk/pull/618/commits/429140f792f0ed9365cf1404b56c8e492130c172)